core/src/test/java/org/elasticsearch/search/aggregations/metrics/TopHitsIT.java

```diff
@@ -1020,6 +1021,18 @@ public void testDontCacheScripts() throws Exception {
         assertThat(client().admin().indices().prepareStats("cache_test_idx").setRequestCache(true).get().getTotal().getRequestCache()
                 .getMissCount(), equalTo(0L));
 
+        // Test that a request using a script sort does not get cached
```

Comment:
Unfortunately not as the rest tests do not install the modules so you can't currently tests scripts in rest tests